crates/rethnet_rpc_server/Cargo.toml

```diff
@@ -4,10 +4,16 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+axum = "0.6.18"
 hashbrown = { version = "0.13", default-features = false, features = ["ahash"] }
+hyper = "0.14.26"
 rethnet_eth = { version = "0.1.0-dev", path = "../rethnet_eth", features = ["serde"] }
+rethnet_evm = { version = "0.1.0-dev", path = "../rethnet_evm" }
+rethnet_evm_napi = { version = "0.1.0-dev", path = "../rethnet_evm_napi" }
```

Comment:
We should not need napi here

Reply:
https://github.com/NomicFoundation/hardhat/pull/3998/commits/394d688b2f2f5e705615ac46e6f5eb6a16147b48

Reply:
in an early iterations i had tried out a direct instantiation of the runtime object in rethnet_evm_napi, but i quickly realized that wasn't right, but this old dependency was left unnoticed. thanks for pointing it out.